### PR TITLE
Feat/get cards

### PR DESCRIPTION
**Pull request by MarcusSchweizer91**
GET all Cards von backend bis frontend

### CODE REVIEW

**Comment by VladimirKlippert93** [comment]
looks fine

File: frontend/src/CardGallery/components/PokemonApp.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }, [])
+
+
+    function getAllCards() {
+        axios.get("/api/cards").then((response) => {
+            setPokeCards(response.data)
+        })
+            .catch(e => console.error(e))
+    }
+
```
Comment (NickKlock) [inline frontend/src/CardGallery/components/PokemonApp.tsx:22]: Maybe you can create a custom hook for all that

File: frontend/src/CardGallery/components/PokemonApp.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    useEffect(() => {
+        getAllCards()
+    }, [])
+
+
+    function getAllCards() {
+        axios.get("/api/cards").then((response) => {
+            setPokeCards(response.data)
+        })
+            .catch(e => console.error(e))
```
Comment (NickKlock) [inline frontend/src/CardGallery/components/PokemonApp.tsx:20]: You can move all your request in an extra file - CLEAN CODE

**Review by MoinMarcell — COMMENTED** [review]
LGTM!

**Pull request merged**